internal/pkg/describe/env.go

```diff
@@ -317,6 +317,39 @@ func (d *EnvDescriber) filterDeployedJobs() ([]*config.Workload, error) {
 	return deployedJobs, nil
 }
 
+// ValidateCFServiceDomainAliases returns error if an environment using cdn is deployed without specifying http.alias for all services
+func (d *EnvDescriber) ValidateCFServiceDomainAliases() error {
+
+	stackDescr, err := d.cfn.Describe()
+	if err != nil {
+		return err
+	}
+
+	services, err := d.deployStore.ListDeployedServices(d.app, d.env.Name)
```

Comment:
Instead of calling this API we might need to grab from the `ALBWorkloads` param from the env parameters. Because `ListDeployedServices` includes services of the other types than just LBWS.